tools/make_class_power.py

```diff
@@ -127,6 +125,10 @@ def make_class_power(paramfile, external_pk = None, extraz=None):
         - Massive neutrinos.
         - Using Sigma8 to set the power spectrum scale.
         - Different transfer functions.
+
+    We use class velocity transfer functions to have accurate initial conditions
+    even on superhorizon scales, and to properly support multiple species.
+
     Not supported:
         - Warm dark matter power spectra.
         - Rescaling with different transfer functions."""
```

Comment:
Rescaling can be done by running the 3-fluid model backward from target redshift to the initial redshift. Caveat is we don't yet know effective sound-speed of baryons. It can be a cool entry level student project.

Reply:
Sure, comment changed. I prefer not to rescale, precisely because it hides force errors like the ones we are encountering :)